services/fhir.ts

```diff
@@ -180,24 +278,49 @@ export async function deleteFHIRResource<R extends AidboxResource>(
         return failure({});
     }
 
-    return service({
+    return service(markAsDeleted(resource));
+}
+
+export function markAsDeleted<R extends AidboxResource>(resource: AidboxReference<R>): AxiosRequestConfig {
+    const inactiveMappingItem = inactiveMapping[resource.resourceType];
+
+    if (!inactiveMappingItem) {
+        console.error(`Please specify inactiveMapping for ${resource.resourceType} to mark item deleted`);
+        return {};
```

Comment:
Let's throw an exception. It is easy to skip the error because everything is success